### ISSUE TITLE
What does reference date mean?

### ISSUE THREAD

**Issue by tombeteende** [synthetichealth/synthea#1511]
### What happened?

There is no documentation about what reference date means, I couldn't trace it in the code or to see how it influences the data I generate (I tried with several reference dates)
(my ultimate goal is to create data that ends in the future and not today)

### Environment

```markdown
- OS: MacOs
- Java:
```

**Comment by dehall** [comment]
Yes I see this can be documented better. The reference date is used to calculate birthdates, such that the ages of the generated population match the age distribution within the target demographics. By default the reference date matches the end date of the simulation which is the current day.

Eg, let's say the age breakdown for a given region is:
    0 – 14 years: 18.2%
    15 – 24 years: 13.0%
    25 – 54 years: 39.0%
    55 – 64 years: 12.9%
    65 years and over: 16.8%

When Synthea runs it tries to meet those %s by picking a target age for each record, then their birthdate is `reference_date - target_age`

**Comment by jawalonoski** [comment]
> (my ultimate goal is to create data that ends in the future and not today)

Also, if you want to end in the future, you should be looking at `[-e endDate as YYYYMMDD]`
